sdk/test/opentelemetry/sdk/trace/span_test.rb

```diff
@@ -202,7 +202,7 @@
     end
 
     it 'honours an explicit timestamp' do
-      ts = Time.new('2021-11-23 12:00:00.000000 -0600')
+      ts = Time.now
```

Comment:
Had to change it due to, so changes to `Time` class introduced in 3.1
```
OpenTelemetry::SDK::Trace::Span::#finish#test_0002_honours an explicit end timestamp:
2022-01-17T14:36:54.7595272Z ArgumentError: invalid value for Integer(): "2021-11-23 12:00:00.000000 -0600"
2022-01-17T14:36:54.7595701Z     <internal:timev>:310:in `initialize'
2022-01-17T14:36:54.7596384Z     /home/runner/work/opentelemetry-ruby/opentelemetry-ruby/sdk/test/opentelemetry/sdk/trace/span_test.rb:394:in `new'
2022-01-17T14:36:54.7597270Z     /home/runner/work/opentelemetry-ruby/opentelemetry-ruby/sdk/test/opentelemetry/sdk/trace/span_test.rb:394:in `block (3 levels) in <top (required)>'```

Reply:
we use static values for time in tests to avoid erratic test results. Please update these to use `Time.parse` instead.

Reply:
@arielvalentin I double checked some of these tests, and I think I (for reasons now unknown to me) parsed explicit timestamps but it didn't actually matter for the tests.  

The check is just looking at whatever we captured from parse (or in this revision time.now) 
`_(events.first.timestamp).must_equal(exportable_timestamp(ts))`

I think this should be ok.